src/vma/dev/cq_mgr.h

```diff
@@ -242,7 +249,9 @@ class cq_mgr
 	const uint32_t		m_n_sysvar_rx_prefetch_bytes_before_poll;
 	const uint32_t		m_n_sysvar_rx_prefetch_bytes;
 	size_t			m_sz_transport_header;
-
+	ib_ctx_handler*		m_p_ib_ctx_handler;
+	const uint32_t		m_n_sysvar_rx_num_wr_to_post_recv;
+	bool			m_skip_dtor;
```

Comment:
Please rename: `m_skip_dtor` --> `m_skip_cleanup`.

#### introduced in:
MP_RQ: add cq_mgr_mp and qp_mgr_mp